html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="/html-relative-links/index.html">Home Page</a></li>
```

Comment:
Hi Timothy, instead of specifying the folder name, you can use '../' to move up one directory. For example: 
href="../index.html"